src/pyodide.js

```diff
@@ -277,6 +290,14 @@ var languagePluginLoader = new Promise((resolve, reject) => {
     return {};
   };
 
+  Module.checkABI = function(ABI_number) {
+    if (ABI_number !== parseInt('{{ABI}}')) {
+      console.error(`ABI numbers differ. Expected {{ABI}}, got ${ABI_number}`);
+      throw `ABI numbers differ. Expected {{ABI}}, got ${ABI_number}`;
```

Comment:
One last comment, then I think this PR is ready.

Can you assign this message to a variable so we don't have to recreate it twice (and run the risk of them becoming out of sync in the future)?

Reply:
Aah yes, good idea! Should I define it in this function aka ``Module.checkABI`` itself?

Reply:
Yes.  It can just be right here.